Documentation/README.md

```diff
@@ -11,15 +11,20 @@ We plan to continue adding support for other storage systems and environments ba
 
 ## Quick Start Guides
 
-Starting Rook in your cluster is as simple as two `kubectl` commands. See our [Quickstart](quickstart-toc.md) guide for the details on what you need to get going.
+Starting Rook in your cluster is as simple as a few `kubectl` commands depending on the storage provider.
+See our [Quickstart](quickstart.md) guide list for the detailed instructions for each storage provider.
 
 ## Storage Provider Designs
 
 High-level Storage Provider design documents:
 
-| Storage Provider | Status | Description |
-|---|---|---|
-| [Ceph](ceph-storage.md) | Stable | Ceph is a highly scalable distributed storage solution for block storage, object storage, and shared file systems with years of production deployments. |
-| [EdgeFS](edgefs-storage.md) | Stable | EdgeFS is high-performance and fault-tolerant object storage system with Geo-Transparent data access to file, block or object. |
+| Storage Provider            | Status | Description                                                                                                                                             |
```

Comment:
Was this table formatting significant? The rendering looks the same to me.

Reply:
It is not too significant, it is more or less just for "cleanliness" of the Markdown tables to look "better" when editing the files.